File: Maximum Identical Bowls.java
class Solution {
    public static int getMaximum(int N, int[] arr) {
        // code here
    long sum = 0;
    for(int i : arr) {
        sum += i;
    }
    for(int i = 0; i < N; i++) {
        if(sum % (N - i) == 0) {
            return N - i;
        }
    }
    return 0;
    }
}

File: Maximum Identical Bowls.cpp
class Solution {
  public:
    int getMaximum(int n, vector<int> &arr) {
        
        long long int sum=0;
        for(auto it:arr)sum+=it;
        for(int i=0;i<n;i++){
            if(sum%(n-i)==0)return n-i;
        }
        return 0;
    }
};
